package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -644,6 +722,13 @@ def _single_frame(self):
         )
         hbond_indices = np.where(d_h_a_angles > self.d_h_a_angle)[0]
 
+        if np.size(hbond_indices) == 0:
```

Comment:
As above, is `np.size` needed?

Reply:
The variable `hbond_indices` is a np.array, so if it is not empty the error "The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()" arises.

Reply:
Thanks for explaining!